sample_app/source/tealium/tealiumBuilder.brs

```diff
@@ -70,6 +70,11 @@ function TealiumBuilder(account as String, profile as String, logLevel as Intege
             return m
         end function
 
+        SetLogLevel: function (logLevel as Integer) as Object
```

Comment:
If a user puts in 4+, looks like system will be ok, will basically be equivalent to 3.  What about a negative integer value?